lib/spack/spack/main.py

```diff
@@ -88,6 +88,7 @@
 
 #: Recorded directory where spack command was originally invoked
 spack_working_dir = None
+spack_ld_library_path = os.environ.get('LD_LIBRARY_PATH', '')
```

Comment:
is `''` better than `None` for some reason?  Just curious.

Reply:
Because then I don't have to check for `None` when I set environment variables with it in other places, since `''` is `None` in the shell where we use this.